frontend/views/pages/Page.vue

```diff
@@ -5,7 +5,7 @@ div(:data-test='pageTestName' :class='$scopedSlots.sidebar ? "p-with-sidebar" :
     h1.is-title-2.p-title(:data-test='pageTestHeaderName' v-if='$slots.title')
       img.c-logo(
         src='/assets/images/group-income-icon-transparent.png'
-        :alt='L("Group Income\'s logo")'
```

Comment:
Is there a reason this was deleted?

Reply:
Yes. This image doesn't add any extra value to the content, it's only decorative. This is a title and having `{{ title }}` is enough. There are no benefits for the SR (screen reader) to read this image to the user, so in this case we just leave an empty alt and the SR will ignore the image.